modules/cli/src/main/scala/scala/cli/commands/package0/Package.scala

```diff
@@ -708,12 +718,41 @@ object Package extends ScalaCommand[PackageOptions] with BuildCommandHelpers {
       .withOsKind(Properties.isWin)
       .callsItself(Properties.isWin)
       .withJavaOpts(build.options.javaOptions.javaOpts.toSeq.map(_.value.value))
-    val params = Parameters.Bootstrap(Seq(loaderContent), mainClass)
+    var params = Parameters.Bootstrap(Seq(loaderContent), mainClass)
       .withDeterministic(true)
       .withPreamble(preamble)
 
+    val params0 =
+      if (build.options.notForBloopOptions.doSetupPython.getOrElse(false)) {
+        val res = value {
+          Artifacts.fetch(
+            Positioned.none(Seq(dep"io.github.alexarchambault.python:interface:0.1.0")),
```

Comment:
The dependency version should be saved in `deps.sc`, this will allow scala-steward to automatically bump this version.

Reply:
Adding it in `deps.sc` isn't enough, but I enabled it, yeah.